### ISSUE TITLE
[Feature] Add inverted "focus" blur

### ISSUE THREAD

**Issue by CurtisBelt** [lupoDharkael/flameshot#165]
* Work just like current blur tool, but invert the effect.
* Handy to give complete focus on the selected area in a single click
* Can currently accomplish this manually with blur tool, but with several carefully aligned blur rectangles.

Example result:

![image](https://user-images.githubusercontent.com/4251760/37259216-808ea9dc-2551-11e8-9919-6b030d544526.png)

**Comment by AndydeCleyre** [comment]
Yes, this would be very useful. In terms of UI, maybe invert while the shift key is pressed and the user is dragging the box?

**Comment by ZYZ64738** [comment]
Something new to this?

instead (or additionally) of blurring a kind of lightbox effekt would be also nice:

![image](https://user-images.githubusercontent.com/9817678/166465525-0074e765-e8e8-4a04-ab1c-401984b1302c.png)

**Comment by borgmanJeremy** [comment]
This needs to wait for plugins
